demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -86,6 +87,41 @@ def teardown_class(cls):
         (INVALID_PLAYBOOK_PATH, PLAYBOOK_TARGET, False, PlaybookValidator)
     ]
 
+    INPUTS_IS_VALID_TEST_PLAYBOOKS = [
+        (VALID_TEST_PLAYBOOK_PATH, True),
+        (INVALID_PLAYBOOK_PATH, False),
+        (INVALID_PLAYBOOK_PATH_FROM_ROOT, True)
+    ]
+
+    @pytest.mark.parametrize('source, answer', INPUTS_IS_VALID_TEST_PLAYBOOKS)
+    def test_validation_of_test_playbooks(self, source, answer):
+        """
```

Comment:
for all tests you added, update the test docstring according to `Given - When - Then` template